x-pack/plugins/security/public/management/management_service.test.ts

```diff
@@ -43,13 +44,20 @@ describe('ManagementService', () => {
         locator: {} as any,
       };
 
+      const uiConfig: ClientConfigType = {
```

Comment:
Can we add unit tests that verify we are responding to these new config properties correctly?

Reply:
Sure, added a test for `setup()` for the case when we set the these configs to false. Let me know if there is another test case that you have in mind.